recipe/0001-fix-makefile.patch

```diff
@@ -0,0 +1,35 @@
+diff -ruN Raven-rev322/Makefile Raven-rev322-patch/Makefile
+--- Raven-rev322/Makefile	2021-04-23 18:48:03.000000000 +0200
++++ Raven-rev322-patch/Makefile	2021-05-12 09:51:59.504952715 +0200
+@@ -12,17 +12,17 @@
+ #            Jan 2019    Alan J. Barton - added OS name to created exe name
+ ###
+ # appname := raven_rev$(shell svnversion -n . | sed -e 's/:/-/g')_$(shell uname -o).exe
+-appname := Raven.exe
++appname := raven
+ 
+-CXX      := g++
+-CXXFLAGS := -Wno-deprecated
++override CXXFLAGS := $(CXXFLAGS) -Wno-deprecated
```

Comment:
I think it's a very good idea to keep the conda-forge `CXXFLAGS` and just add to them rather than replacing them.  I don't know if this *really* matters but I think it could.